lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -373,6 +374,112 @@
+
+    entrust_api_user:
+        description:
+            - The username for authentication to the Entrust ECS API.
+        type: str
+        version_added: "2.9"
+
+    entrust_api_key:
+        description:
+            - The key (password) for authentication to the Entrust ECS API.
+        type: str
+        version_added: "2.9"
+
+    entrust_api_client_cert_path:
+        description:
+            - The path of the client certificate used to authenticate to the Entrust ECS API.
+        type: path
+        version_added: "2.9"
+
+    entrust_api_client_cert_key_path:
+        description:
+            - The path of the key for the client certificate used to authenticate to the Entrust ECS API
+        type: path
+        version_added: "2.9"
+
+    entrust_reissue_if_expires_in:
+        description:
+            - If this value is set and an existing certificate is present, it will be reissued if it expires within this number of seconds.
+            - If not present or value is 0, it will be reissued if the certificate is already expired.
+            - For example, if you want to reissue a certificate if it is expiring within a month, set this to 2592000.
+              Certificates with more seconds remainingpast this number of seconds from when the module is called will be left as is.
+        type: int
+        version_added: "2.9"
+
+    entrust_not_after:
+        description:
+            - The point in time at which the certificate stops being valid.
+            - Time can be specified either as relative time or as absolute timestamp.
+            - Time will always be interpreted as UTC.
+            - Valid format is C([+-]timespec | ASN.1 TIME) where timespec can be an integer
+              + C([w | d | h | m | s]) (e.g. C(+32w1d2h).
+            - Note that if using relative time this module is NOT idempotent.
+            - If this value is not specified, the certificate will stop being valid 365 days from now.
+        type: str
+        default: +365d
+        version_added: "2.9"
+
+    entrust_eku:
+        description:
+            - The extended key usage requested for the certificate.
+        choices: ['SERVER_AUTH', 'CLIENT_AUTH', 'SERVER_AND_CLIENT_AUTH']
+        type: str
+        default: SERVER_AND_CLIENT_AUTH
+        version_added: "2.9"
+
+    entrust_api_specification_path:
+        description:
+            - Path to the specification file defining the Entrust ECS api.
+            - Can be used to keep a local copy of the specification to avoid downloading it every time the module is used.
+        type: path
+        default: https://cloud.entrust.net/EntrustCloud/documentation/cms-api-2.1.0.yaml
+        version_added: "2.9"
+
+    validate_ocsp:
+        description:
+            - This is currently supported only  when provider is entrust.
+            - If set to True, this module will perform an OCSP check on a certificate before indicating that the state is present and certificate is valid.
```

Comment:
```suggestion
            - If set to C(yes), this module will perform an OCSP check on a certificate before indicating that the state is present and certificate is valid.
```
In general, use `true`/`false` or `yes`/`no` in module documentation. Since other parts of the docs for this module already use `yes`/`no`, please alse use that.